test/api/manipulation.js

```diff
@@ -496,6 +496,24 @@ describe('$(...)', function() {
     });
   });
 
+  describe('.appendTo', function() {
+
+    it('(Node) : should add element as last child', function() {
```

Comment:
The test description is incorrect. It describes a Node-accepting signature, but the test invokes `appendTo` with a Cheerio instance (which is elsewhere described as `"$(...)"`)